release-content/0.14/release-notes/12013_Upstreaming_bevy_color.md

```diff
@@ -1 +1,31 @@
-TODO
+Colors are a huge part of building a good game: UI, effects, shaders and more all need fully-feature, correct and convenient color tools.
+Bevy now supports a wide range of color spaces, each with their own type (e.g. `LinearRgba`, `Hsla` or `Oklaba`),
+and offers a wide range of fully documented operations on and conversions between them.
+
+Critically, color mixing / blending is now supported: perfect for procedurally generating color palettes and working with animations.
```

Comment:
I don't know if we want to mention this, but the bevy_color can offer better performance in some cases, and is more idiomatic Rust.

Reply:
Better performance I think is a side effect to the real win here in my opinion, which is we're able to give the user more control and transparency over how colours are represented. If a Bevy API wants a `LinearRgba` colour, now the user knows that and can adjust their algorithms accordingly (if they want to!).

Reply:
Let's just say that I think we should do a better job of listing the benefits of bevy_color; after all, we're asking the users to do a fairly significant migration, so we ought to "sell" them on the advantages.